Formula/geode.rb

```diff
@@ -0,0 +1,36 @@
+class Geode < Formula
+  desc "In-memory Data Grid for fast transactional data processing"
+  homepage "https://geode.apache.org"
+  url "http://apache.org/dyn/closer.cgi?filename=incubator/geode/1.0.0-incubating.M2/apache-geode-1.0.0-incubating.M2.tar.gz&action=download"
+  version "1.0.0-incubating.M2"
```

Comment:
Yes, this is official release approved by the Apache Incubator PMC.  Here's the link to the announcement:

http://mail-archives.apache.org/mod_mbox/incubator-general/201604.mbox/%3cCAFh+7k2eiK2TMGKsLqrY9CZDjxjYwiuTQ4QGUVC2s3geyJYwnA@mail.gmail.com%3e